dbt/include/global_project/macros/relations/column/columns_spec_ddl.sql

```diff
@@ -87,3 +87,11 @@
   {% set formatted = column.column.lower() ~ " " ~ data_type %}
   {{ return({'name': column.name, 'data_type': data_type, 'formatted': formatted}) }}
 {%- endmacro -%}
+
+{% macro render_unspecified_column_value(column_type) %}
+  {{ return(adapter.dispatch('render_unspecified_column_value', 'dbt') (column_type)) }}
+{% endmacro %}
+
+{% macro default__render_unspecified_column_value(column_type) %}
+    safe_cast('null', {{column_type}})
+{% endmacro %}
```

Comment:
As the input here may actually be 'specified' I think this needs to be a more generic name
```suggestion
{% macro render_column_type(column_type) %}
  {{ return(adapter.dispatch('render_column_type', 'dbt') (column_type)) }}
{% endmacro %}

{% macro default__render_column_type(column_type) %}
    safe_cast('null', {{column_type}})
{% endmacro %}
```